src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/CourtCaseEventsServiceTest.kt

```diff
@@ -74,11 +69,11 @@ class CourtCaseEventsServiceTest {
   @Test
   fun `should call telemetry service when exact match found for Court Case Event`() {
     // Given
-    val pncNumber = "PNC12345"
+    val pncNumber = "20030011985X"
```

Comment:
Had to do this as real validation is now happening - I borrowed this ID from the PNCIdValidatorTest